packages/react/src/ComparisonTable/ComparisonTable.module.css

```diff
@@ -7,8 +7,9 @@
 
 @media screen and (min-width: 34rem) {
   .ComparisonTable--heading {
+    max-width: var(--brand-ComparisonTable-heading-maxWidth);
     text-align: center;
-    margin-bottom: var(--base-size-40);
+    margin-bottom: var(--base-size-96);
```

Comment:
@egim is this update specifically to address Resources Hub or will it also apply to all GitHub websites and their designs in future? 

If this is likely to change from site to site, I think this should be a design token to enable customisation (e.g. --brand-ComparisonTable-heading-marginBottom). 

We'll also need to update the Figma component too if it goes ahead. cc. @danielguillan @josepmartins

Reply:
> @egim is this update specifically to address Resources Hub or will it also apply to all GitHub websites and their designs in future?

I asked this question [here](https://github.com/github/marketing-engineering/issues/1397#issuecomment-1516946233). Looking for some advice from maybe @ajashams or @jesussandreas 😄. 

> If this is likely to change from site to site, I think this should be a design token to enable customisation (e.g. --brand-ComparisonTable-heading-marginBottom).

Makes sense to me!

Reply:
i think it needs to happen globally, but we'll need to define spacing for that. so in my eyes. this is kinda of a temporary fix until we get to that point.